README.md

```diff
@@ -0,0 +1,37 @@
+# TEMP_TITLE
+
+## Organizacja plików i sprawdzanie rozwiązań
+W katalogu **WDI** należy znaleźć interesujący nas zestaw oraz zadanie. Następnie w tym katalogu umieścić kod w pliku o nazwie **prog.py**
+i uruchomić plik core.py. Wynik sprawdzenia znajdzie się w pliku DATA.txt w katalogu z zadaniem.
+
+Przykładowo aby sprawdzić zadanie 7 z zestawu 2 należy w pliku **./WDI/Set_2/Task_7/prog.py** umieścić kod, a następnie w pliku po uruchomieniu
+**core.py** w pliku **./WDI/Set_2/Task_7/DATA.txt** pojawi się aktualna punktacja za zadanie.
+
+Sprawdzić można jednocześnie wiele zadań. Jeżeli plik **prog.py** jest nowszy niż **DATA.txt**, to **core.py** automatycznie sprawdzi wszystkie 
+zaktualizowane wersje dodanych rozwiązań zadań.

```

Comment:
It's a bit hard to understand 😅
LMKWYT about creating a downloadable CLI and providing some `sh` commands with options.

Reply:
It may be added in later but I think it's a great thing to aim for

Reply:
Or a bit earlier but not now